modules/regional-go-service/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "cloud_run_endpoint" {
+  description = "Cloud Run services endpoint"
+  value       = zipmap(values(google_cloud_run_v2_service.this).*.location, values(google_cloud_run_v2_service.this).*.uri)
+}
```

Comment:
While it's annoying, I had specifically avoided exposing this to ~force folks to go through `authorize-private-service`, and I'd kind of like to keep that.

However, you should be able to do something similar to that to look it up once this has deployed:
https://github.com/chainguard-dev/terraform-infra-common/blob/7112a5ef4814be6321fed4ef75f85ba54645bd37/modules/authorize-private-service/main.tf#L10-L16

Reply:
got it, i will drop that

Reply:
@mattmoor ptal